### DISCUSSION TITLE
v1.0.0

### DISCUSSION THREAD

**Discussion by ehsandeep** [projectdiscovery/interactsh#199] [Releases]
## What's Changed

* Added multiple server support + random pickup to client by @Ice3man543 in #172
* Added new pool of interactsh server to use by @ehsandeep in #177
* Added certmagic library support by @Ice3man543 in #162
* Added `goburrow/cache` for internal kv store by @Ice3man543 in #160
* Added TCP socket for DNS Server by @fincham in #182
* Added ACME/TLS optional support (-skip-acme) by @Mzack9999 in #149
* Added LDAP Server support by @Mzack9999, @maikthulhu in #55
* Added HTTP fallback support on HTTPS failure by @Mzack9999 in #154
* Added support for FTP service (-ftp) by @Mzack9999 in #104
* Added custom port support by @kalidor in #128
* Added server listening service listing by @AV-IO in #130
* Added goflags support by @Mzack9999 in #183
* Fixed multiple bugfixes in DNS server by @Ice3man543 in #162
* Fixed netifaces pip package in Dockerfile by @kalidor in #143
* Fixed case insensitive domain name (#75) by @coffeemakr in #121
* Fixed lowercase FQDNs in responses by @Serizao in #142
* Removed app cname entry by @Mzack9999 in #155

## New Contributors

* @coffeemakr made their first contribution in #121
* @random-robbie made their first contribution in #123
* @kalidor made their first contribution in #128
* @AV-IO made their first contribution in #130
* @maikthulhu made their first contribution in #131
* @Serizao made their first contribution in #142
* @zt2 made their first contribution in #161
* @fincham made their first contribution in #182

**Full Changelog**: https://github.com/projectdiscovery/interactsh/compare/v0.0.7...v1.0.0

<hr /><em>This discussion was created from the release <a href='https://github.com/projectdiscovery/interactsh/releases/tag/v1.0.0'>v1.0.0</a>.</em>

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
